data/schema/schema_v1.json

```diff
@@ -471,10 +469,12 @@
                               "type": "object",
                               "properties": {
                                 "show_guidance": {
-                                  "type": "string"
+                                  "type": "string",
+                                  "description": "Enables guidance on how to answer the question to be displayed."
                                 },
                                 "hide_guidance": {
-                                  "type": "string"
+                                  "type": "string",
+                                  "description": "Enables guidance on how to answer the question to be hidden."
```

Comment:
This property is the text that is used for the 'Hide guidance' link

Reply:
done